src/components/input-number/input-number.stories.ts

```diff
@@ -121,3 +121,6 @@ export const hebrewNumberingSystem_TestOnly = (): string =>
 
 export const arabicLocaleWithLatinNumberingSystem_TestOnly = (): string =>
   html`<calcite-input-number lang="ar-EG" numbering-system="latn" value="123456"></calcite-input-number>`;
+
+export const mediumIconForLargeInputNumber_TestOnly = (): string =>
+  html`<calcite-input type="number" placeholder="Placeholder" scale="l" value="123" step="1"></calcite-input>`;
```

Comment:
Same here, add `scale="l"` to one of the numberingSystem or locale stories. Also this should be `calcite-input-number` instead of `calcite-input type="number"`

Reply:
This component doesn't use `calcite-input` internally, so we have to update `input`, `input-number`, and `input-text` separately for now. There is an issue for creating shared utils.